src/AddressRegistry/StreetName/ValueObjects/PositionSpecification.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace AddressRegistry.StreetName
+{
+    public enum PositionSpecification
```

Comment:
There is already an enum `GeometrySpecification`

Reply:
Should I remove from it the values Street, BuildingUnit and Building? https://github.com/Informatievlaanderen/address-registry/blob/main/src/AddressRegistry/StreetName/ValueObjects/GeometrySpecification.cs

Reply:
Leave them untouched as they are part of the domain. Only new addresses won't be able to use some of the old specifications.